v2/internal/frontend/desktop/windows/browser.go

```diff
@@ -5,10 +5,32 @@ package windows
 
 import (
 	"github.com/pkg/browser"
+	"golang.org/x/sys/windows"
 )
 
+var fallbackBrowserPaths = []string{
+	`\Program Files (x86)\Microsoft\Edge\Application\msedge.exe`,
+	`\Program Files\Google\Chrome\Application\chrome.exe`,
+	`\Program Files (x86)\Google\Chrome\Application\chrome.exe`,
+	`\Program Files\Mozilla Firefox\firefox.exe`,
+}
+
 // BrowserOpenURL Use the default browser to open the url
 func (f *Frontend) BrowserOpenURL(url string) {
 	// Specific method implementation
-	_ = browser.OpenURL(url)
+	err := browser.OpenURL(url)
+	if err == nil {
+		return
+	}
+	for _, fallback := range fallbackBrowserPaths {
+		if err2 := openBrowser(fallback, url); err2 == nil {
+			f.logger.Warning("default browser.OpenURL failed but fell back to %v, original err: %v", fallback, err.Error())
```

Comment:
I'm not sure we need these. I'm wondering what sort of action would be taken based on seeing them.

Reply:
I agree, removed in https://github.com/wailsapp/wails/compare/69375ca82904e0c32489f98ff6db6eb05ae542e8..cdeac67d3dc53df24178b92666fbb6b4b090d3ce`